### ISSUE TITLE
py_mob func iso_bin, IndexError For List.

### ISSUE THREAD

**Issue by IvanaXu** [statcompute/py_mob#2]
https://github.com/statcompute/py_mob/blob/91ff2a29235cef4d3ce806d8a5e5857c237de44e/py_mob/py_mob.py#L393-L427

``` python
 _p = sorted([_[1] for _ in [l for l in _l3 if l[2] < 1 and l[2] > 0 and l[3] > 10]])[1:-1] 
```
if sorted([_[1] for _ in [l for l in _l3 if l[2] < 1 and l[2] > 0 and l[3] > 10]]) is short than 3
just like [0, 1], cut [1:-1] to _p will be []

``` python
_["rule"] = "$X$ > " + str(_p[-1]) 
```
or 
``` python
_["rule"] = "$X$ <= " + str(_p[0])
```
will be 
```
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
IndexError: list index out of range
```

**Comment by statcompute** [comment]
try below if it can fix the issue:
_p = sorted([_[1] for _ in [l for l in _l3 if l[2] < 1 and l[2] > 0 and l[3] > 3]])[:-1]

Thanks

**Comment by statcompute** [comment]
The issue should have been fixed in the version 0.2.1 published on 05/23/2020.
